### PR TITLE
feat: add country to nodes

### PR DESCRIPTION
**Pull request by Cifko**
Review #60 first, and after merge this can be reviewed.

Add country column to nodes. So we can have a distribution in the dashboard.

### CODE REVIEW

File: atoma-state/src/types.rs [hunk 1]
```diff
@@ -115,6 +115,13 @@ pub struct CheapestNode {
     pub max_num_compute_units: i64,
 }
 
+/// Response for getting the node distribution
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, FromRow)]
```
Comment (jorgeantonio21) [inline atoma-state/src/types.rs:119]: Maybe expand on the docs here

File: atoma-state/src/state_manager.rs [hunk 2]
```diff
@@ -619,6 +633,74 @@ impl AtomaState {
...
+    /// # Example
+    ///
+    /// ```rust,ignore
+    /// use atoma_node::atoma_state::AtomaStateManager;
+    ///
+    /// async fn get_nodes_distribution(state_manager: &AtomaStateManager) -> Result<Vec<(String, usize)>, AtomaStateManagerError> {
+    ///    state_manager.get_nodes_distribution().await
+    /// }
+    /// ```
+    pub async fn get_nodes_distribution(&self) -> Result<Vec<NodeDistribution>> {
```
Comment (jorgeantonio21) [inline atoma-state/src/state_manager.rs:691]: add instrumentation

**Review by jorgeantonio21 — COMMENTED** [review]
LGTM, let's first merge the previous PR, address the comments and then merge

File: atoma-state/src/types.rs (hunk 1 above)
Comment (Cifko) [reply atoma-state/src/types.rs:121]: check the new comment, and let me know what you think

Comment (jorgeantonio21) [reply atoma-state/src/types.rs:121]: Much better honestly

**Review by jorgeantonio21 — APPROVED** [review]

**Review by jorgeantonio21 — APPROVED** [review]
LGTM

**Pull request merged**